Expecto.Tests/Tests.fs

```diff
@@ -226,6 +226,68 @@ let timeouts =
       testCase "list" <| fun _ ->
         let _, isList = ExpectoConfig.fillFromArgs defaultConfig [|"--list-tests"|]
         isList ==? true
+
+      testList "filtering" [
+        let dummy =
+          TestList ([
+                      testCase "a" ignore
```

Comment:
Oh sure, was just thinking you want to use this style since it was used also in https://github.com/haf/expecto/blob/master/Expecto.Tests/Tests.fs#L158.

Will fix both.